lib/util/markdown.ts

```diff
@@ -18,6 +18,10 @@ export function sanitizeMarkdown(markdown: string): string {
   const backTickRe = regEx(/&#x60;([^/]*?)&#x60;/g);
   res = res.replace(backTickRe, '`$1`');
   res = res.replace(regEx(/`#&#8203;(\d+)`/g), '`#$1`');
+  res = res.replace(
+    regEx(/(?<before>[^\n]\n)(?<title>####.*)/g),
```

Comment:
shouldn't we allow any header level?
```suggestion
    regEx(/(?<before>[^\n]\n)(?<title>#.*)/g),
```